src/pages/index.js

```diff
@@ -28,6 +30,19 @@ const cardSection = new Section(
 const popupWithImage = new PopupWithImage("#preview-modal");
 popupWithImage.setEventListeners();
 
+const confirmPopup = new ConfirmPopup("#confirmationModal");
+
+const profileImage = document.querySelector(".profile__image");
+
+const profileImageModal = new PopupWithForm(
+  "#profile-image-modal",
+  handleProfileImageFormSubmit
+);
+
+profileImage.addEventListener("click", () => {
+  profileImageModal.open();
+});
+
 const config = {

```

Comment:
From the 8th sprint all constants (including `initialCards` and `config `) should be in `constants.js` in the `utils` folder to clean up the main file.